docs/4-schema-design.md

```diff
@@ -267,26 +269,10 @@ Now that we have reorganized our mutations, we will refactor the schema to a pro
     >    .LoadAsync(id, ctx.RequestAborted));
     > ```
 
-1. Head over to the `Query.cs` and annotate the `id` argument of `GetSpeaker` with the `ID` attribute.
```

Comment:
I ended up removing this part because the ID attribute expects a hash code as a result. This was causing problems running the queries provided in the examples. Removing this annotation in the following parts of the code made all the queries work because they use integers for ID.